mmdt/survey/migrations/0001_initial.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.4 on 2024-02-25 12:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SurveyForm',
+            fields=[
+                ('form_name', models.CharField(default='DefaultFormName', max_length=255)),
+                ('form_id', models.AutoField(primary_key=True, serialize=False)),
+                ('is_active', models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SurveyQuestion',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('question_name', models.CharField(max_length=255)),
+                ('pub_date', models.DateTimeField(verbose_name='date published')),
+                ('is_enabled', models.BooleanField(default=True)),
+                ('survey_group', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='questions', to='survey.surveyform')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Response',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.TextField()),
```

Comment:
As I commented in here , https://vigorzwe.slack.com/archives/C06EWC7BA8L/p1708561667497389?thread_ts=1708533249.637779&cid=C06EWC7BA8L Survey question types may be several options not only text data. Shall we add 
1. multiple choice
2. checkbox
3. text box
4. long text box 
5. Others 
Question types?? It will be the best way to collect the survey data. When we collect the survey, we will not collect only text.

Reply:
Sorry for the misunderstanding ama. I will fix and make a new commit.